docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,231 @@ sidebar_position: 60
+| Provisioning          | Allow disk access<br />Allow file access<br />Allow read-only disk access<br />Allow virtual machine download<br />Allow virtual machine files upload<br />Clone template<br />Clone virtual machine<br />Create template from virtual machine<br />Customize guest<br />Deploy template<br />Mark as template<br />Mark as virtual machine<br />Modify customization specification<br />Promote disks<br />Read customization specifications                                                                                                                                                                                                                                         |
+| Service Configuration | Allow notifications<br />Allow polling of global event notifications<br />Manage service configurations<br />Modify service configuration<br />Query service configurations<br />Read service configuration                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| Snapshot Management   | Create snapshot<br />Remove snapshot<br />Rename snapshot<br />Revert to snapshot                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| vSphere Replication   | Configure replication<br />Manage replication<br />Monitor replication                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I don't understand what this table is compared to the previous one?

Reply:
The _Spectro role_ tables are actually all identical. We just have three of them to support the tabbed "version selection".